test/unit/server/getters/get-platform-activity-stats.js

```diff
@@ -0,0 +1,33 @@
+const { setupTestDb, seedDb } = require("test.database");
```

Comment:
Mind if we do tests in TypeScript, too?

Reply:
Wow. I thought jest didn't support that but it appears I'm wrong.

I can migrate the tests to ts on the followup PRs that will have the knex stuff sorted out.